tests/src/core/Entity.test.ts

```
@@ -14,20 +14,40 @@ describe("Entity", async () => {
+
+      expect(scene.findEntityByPath("root")).eq(root);
 
-      expect(scene.findEntityByPath("root/parent")).eq(parent);
+      expect(scene.findEntityByPath("root/parent")).eq(parentX);
 
+      expect(scene.findEntityByPath("root/parent/null")).eq(null);
+
+      expect(scene.findEntityByPath("root/parent/child1")).eq(child1);
       expect(scene.findEntityByPath("root/parent/child")).eq(child);
+      expect(scene.findEntityByPath("root/parent/child2")).eq(child2);
+      debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)